pkgs/by-name/po/postgresqlTestHook/postgresql-test-hook.sh

```diff
@@ -1,8 +1,9 @@
-preCheckHooks+=('postgresqlStart')
-postCheckHooks+=('postgresqlStop')
+preCheckHooks+=(postgresqlStart)
+postCheckHooks+=(postgresqlStop)
 
 
 postgresqlStart() {
+  PATH="@postgresqlDev@/bin:@postgresql@/bin:$PATH"
```

Comment:
Using `propagatedBuildInputs` from the setup hook should also work in principle, shouldn't it? This would alleviate the burden of reimplementing existing logic here.